UI/src/asset-type-layer-specifications.js

```diff
@@ -621,6 +631,78 @@
        // check if administrative class is State
           return selectedAsset.getAdministrativeClass(linkId) === "State";
      }
+    },
+    {
+      typeId: assetType.trHeightLimits,
+      layerName: 'trHeightLimits',
+      title: 'TR suurin sallittu korkeus',
+      allowComplementaryLinks: true,
+      allowGrouping: true,
+      groupingDistance: 25 ,
+      legendValues: [
+        {symbolUrl: 'images/point-assets/point_blue.svg', label: 'Rajoitus'},
+        {symbolUrl: 'images/point-assets/point_red.svg', label: 'Geometrian ulkopuolella'}
+      ],
+      formLabels: {
+        title: 'Rajoitus',
+        showUnit: true
+      },
+      editConstrains : function() {
+        return true;
+      },
+      nonModifiableBox: true,
+      label: new HeightLimitLabel()
+    },
+    {
+      typeId: assetType.trWidthLimits,
+      layerName: 'trWidthLimits',
+      title: 'TR suurin sallittu leveys',
+      allowComplementaryLinks: true,
+      allowGrouping: true,
+      groupingDistance: 25,
+      legendValues: [
+        {symbolUrl: 'images/point-assets/point_blue.svg', label: 'Rajoitus'},
+        {symbolUrl: 'images/point-assets/point_red.svg', label: 'Geometrian ulkopuolella'}
+      ],
+      formLabels: {
+        title: 'Rajoitus',
+        showUnit: true
+      },
+      editConstrains : function() {
+        return true;
+      },
+      nonModifiableBox: true,
+      label: new WidthLimitLabel()
+    }
+
+  ];
+
+  root.groupedPointAssetSpecs = [
+    {
+      typeIds: assetGroups.trWeightGroup,
+      layerName: 'trWeightLimits',
+      title: 'TR painorajoitukset',
+      allowComplementaryLinks: true,
+      allowGrouping: false,
+      legendValues: [
+        {symbolUrl: 'images/point-assets/point_blue.svg', label: 'Rajoitus'},
+        {symbolUrl: 'images/point-assets/point_red.svg', label: 'Geometrian ulkopuolella'}
+      ],
+      formLabels: {
+        title: 'Painorajoitus',
+        showUnit: true
+      },
+      editConstrains : function() {
+        return true;
+      },
+      nonModifiableBox: true,
+      label: new WeightLimitLabel(),
+      propertyData: [
+        {'propertyTypeId': 320, 'propertyType': 'number', 'publicId': "suurin_sallittu_massa_mittarajoitus", values: []},
+        {'propertyTypeId': 330, 'propertyType': 'number', 'publicId': "yhdistelman_suurin_sallittu_massa", values: []},
+        {'propertyTypeId': 340, 'propertyType': 'number', 'publicId': "suurin_sallittu_akselimassa", values: []},
+        {'propertyTypeId': 350, 'propertyType': 'number', 'publicId': "suurin_sallittu_telimassa", values: []}
```

Comment:
Id values here could be replaced by assetType.trWeightLimits, assetType.trTrailerTruckWeightLimits etc.